src/dndbeyond/content-scripts/feat.js

```diff
@@ -0,0 +1,98 @@
+console.log("Beyond20: D&D Beyond Feats module loaded.");
+
+class FeatCharacter extends CharacterBase {
+    constructor(global_settings) {
+        super("feat", global_settings);
+    }
+    getDict() {
+        const dict = super.getDict();
+        if (this.avatar) {
+            dict.avatar = this.avatar;
+        }
+        return dict;
+    }
+}
+
+var character = null;
+var settings = getDefaultSettings();
+
+function addDisplayFeatButton() {
+    const icon32 = chrome.runtime.getURL("images/icons/badges/normal32.png");
+    const button = E.a({ class: "ct-beyond20-roll button-alt", href: "#" },
+        E.span({ class: "label" },
+            E.img({ class: "ct-beyond20-item-icon", src: icon32, style: "margin-right: 10px;" }),
+            "Display Feat on VTT")
+    );
+    const feat_name = $(".page-title").text().trim();
+    const feat_type = "";
+    const description = descriptionToString(".item-details .more-info-content, .details-container-feat .details-container-content-description").trim();
+    const feat_tags = $(".details-container-content-footer .tags .tag").toArray().map(elem => elem.textContent);
+    $(".page-heading__content").after(button);
+
+    $(".ct-beyond20-roll").css({
+        "float": "right",
+        "display": "inline-block"
+    });
+    $(".ct-beyond20-roll").on('click', (event) => {
+        sendRoll(character, "item", "0", {
```

Comment:
I think the type of roll should be "feature", rather than "item". See `displayFeature` in the `content-scripts/character.js`